operators/ip_match_from_dataset.go

```diff
@@ -10,28 +10,17 @@ import (
 	"github.com/corazawaf/coraza/v3/rules"
 )
 
-type ipMatchFromDataset struct {
```

Comment:
Note how operators like this or PM got simplified by only customizing the factory but not defining a type